src/components/common/InputBox.tsx

```diff
@@ -0,0 +1,51 @@
+import { COLORS } from "constants/color";
+import React from "react";
+import styled from "styled-components";
+import { justifyContent } from "types";
+
+interface InputBoxComponent {
+  children?: React.ReactNode;
+  width?: string;
+  color?: string;
+  backgroundColor?: string;
+  justifyContent?: justifyContent;
+  padding?: string;
+}
+
+export const InputBox = ({
+  children,
+  width,
+  color,
+  backgroundColor,
+  justifyContent,
+  padding,
+}: InputBoxComponent) => {
+  return (
+    <InputBoxStyle
+      width={width}
+      color={color}
+      backgroundColor={backgroundColor}
+      justifyContent={justifyContent}
+      padding={padding}
+    >
+      {children}
+    </InputBoxStyle>
+  );
+};
+
```

Comment:
모든 Property 가 다 optional 이군요..!
만약 정말로 진짜로 최종으로 Props에 required 가 추가되지 않는다는게 보장된다면 
Partial <- 이라는 간지나는 녀석을 쓸 타이밍인것 같습니다.

```suggestion
interface InputBoxComponent {
  children: React.ReactNode;
  width: string;
  color: string;
  backgroundColor: string;
  justifyContent: justifyContent;
  padding: string;
}

export const InputBox = ({
  children,
  width,
  color,
  backgroundColor,
  justifyContent,
  padding,
}: Partial<InputBoxComponent>) => {
  return (
    <InputBoxStyle
      width={width}
      color={color}
      backgroundColor={backgroundColor}
      justifyContent={justifyContent}
      padding={padding}
    >
      {children}
    </InputBoxStyle>
  );
};

```

https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
모든 optional interface를 가진 컴포넌트에 Partial을 적용해주었어요!
물음표가 사라지니까 속이 다 시원합니다 😎😎😎

- [refactor: optional interface에 Partial 적용](https://github.com/usageness/react-payments/pull/1/commits/8b27d01a4e14f44213630d6918b5ea1fad2f9aa4)

Reply:
아주 속이 시원하시죠?